Penumbra/UI/SettingsInterface.cs

```diff
@@ -548,24 +650,34 @@ void DrawInstalledMods()
                         _selectedMod.Enabled = enabled;
                         _plugin.ModManager.Mods.Save();
                         _plugin.ModManager.CalculateEffectiveFileList();
-                    }
-
-                    if( ImGui.Button( "Open Mod Folder" ) )
-                    {
-                        Process.Start( _selectedMod.Mod.ModBasePath.FullName );
-                    }
+                    }
+
+                    DrawEditButtons();
+
+                    DrawGroupSelectors();
+
+                    ImGui.TextWrapped( _selectedMod.Mod.Meta.Description ?? "" );
+
+                    ImGui.BeginTabBar( "PenumbraPluginDetails" );
+                    if ( (_selectedMod.Mod.Meta.ChangedItems?.Count ?? 0 ) > 0)

```

Comment:
not sure if i'm blind or not, but changeditems doesn't seem to be populated from anywhere? there's the list in the meta and thats it - unless mod makers are supposed to fill this?

Reply:
by changed items do you mean like game items that a mod changes or?

Reply:
Yeah, filled by hand in the json. I don't know if there's a way to correlate changed resources to items?
I was basically editing descriptions of mods to include which items they change and thought that it would work better as a list than in the description.

Reply:
yeah there is, in the item sheet there's a model field which is what the game uses to generate a path to the model and load it, which applies to gear and clothing, though the lines get blurred a bit when it comes to racial specific replacements, its still doable without a huge amount of effort